go/signedexchange/cmd/gen-signedexchange/main.go

```diff
@@ -47,6 +47,8 @@ var (
 	flagDumpHeadersCbor      = flag.String("dumpHeadersCbor", "", "Dump metadata and headers encoded as a canonical CBOR to a file for debugging.")
 	flagOutput               = flag.String("o", "out.sxg", "Signed exchange output file. If value is '-', sxg is written to stdout.")
 
+	flagSkipCheck = flag.Bool("skipcheck", false, "Do not reject invalid input arguments")
```

Comment:
bikeshedding: How about `-ignoreErrors` (aligned w/ `make`) or `-f` (most of coreutils)

Reply:
OK, renamed to `-ignoreErrors`.